Source/dead.h

```diff
@@ -5,8 +5,12 @@
  */
 #pragma once
 
+#include <SDL.h>

```

Comment:
Is this just for the `Uint8`? Would probably be better to switch that `Uint8` to `std::uint8_t` instead. Anyway, it's a minor thing.

Reply:
Yeah, the good news is that this makes it clear what files this should be cleaned up in :) I decided to do so at a later point.